academic/import_bibtex.py

```diff
@@ -103,7 +105,12 @@ def parse_bibtex_entry(
         log.error(f'Invalid date for entry `{entry["ID"]}`.')
 
     page.fm["date"] = "-".join([year, month, day])
-    page.fm["publishDate"] = timestamp
+    if "publishDate" in page.fm:
+        pass  # don't overwrite the existing publishDate
+    elif publish_date_from_bibtex:
```

Comment:
The reason we have both `date` and `publishDate` for publications is that some users want to display a future `date` (upcoming publication) but publish the publication page today. This logic would mean their pages may no longer be published right?